src/en/charms-relations.md

```diff
@@ -49,19 +50,27 @@ For example, in the case of specifying a database for the Mediawiki charm.
 
 ```bash
 juju add-relation mediawiki mysql
-error: ambiguous relation: "mediawiki mysql" could refer to 
+error: ambiguous relation: "mediawiki mysql" could refer to
   "mediawiki:db mysql:db"; "mediawiki:slave mysql:db"
 ```
 
-The solution is to specify the nature of the relation using the
-relation interface identifier. In this case, we want MySQL to provide the 
-backend database for mediawiki ('db' relation), so this is what we need to 
-enter:
+The solution is to be explicit when referring to an *endpoint*, where the
```

Comment:
If we are going to start calling these objects 'endpoints', I think we should actually introduce them a bit more than that

Reply:
Right, I made this change minimal because it's not strictly a CMR thing, but it really needed changing. I didn't want to get caught in a rabbit hole. You want more of this here or do you want this done in another PR?

Reply:
Maybe we should tack this on in a different pr when we have nailed down the CMR content

Reply:
OK. I've left this text commented.